owncloudApp/src/main/java/com/owncloud/android/shares/db/OCShare.kt

```diff
@@ -66,9 +71,34 @@ data class OCShare(
     val name: String?,
     @ColumnInfo(name = ProviderTableMeta.OCSHARES_URL)
     val shareLink: String?
-) {
+) : Parcelable {
```

Comment:
Why don't you use @Parcelize?

https://kotlinlang.org/docs/tutorials/android-plugin.html

Reply:
Didn't know it, thanks for the suggestion!

Reply:
But `@Parcelable` is [still experimental](https://kotlinlang.org/docs/tutorials/android-plugin.html#parcelable-implementations-generator), I would not use it for production apps, too brave :grimacing:

Reply:
I will be stable in the coming release...

BTW: 🐔  😄